+++ a/src/wl/sys/wl_linux.c	2015-09-05 16:45:03.797474672 +0200
+++ b/src/wl/sys/wl_linux.c	2015-09-05 16:44:37.390943551 +0200
@@ -205,7 +205,7 @@ module_param(macaddr, charp, S_IRUGO);
 
 static int nompc = 0;
 module_param(nompc, int, 0);
-MODULE_LICENSE("Mixed/Proprietary");
+MODULE_LICENSE("GPL\0Mixed/Proprietary");
 
 #ifdef quote_str
 #undef quote_str
